Client/mods/deathmatch/logic/CRadarMap.h

```diff
@@ -62,6 +62,8 @@ class CRadarMap
     void ZoomIn();
     void ZoomOut();
 
+    void SetMapImage(std::uint32_t imageIndex);
```

Comment:
Can this function be noexcept?

Reply:
idk, continue discussion in https://github.com/multitheftauto/mtasa-blue/pull/3823#discussion_r1821631592